database/factories/ModelFactory.php

```diff
@@ -11,29 +15,19 @@
 |
 */
 
-$factory->define(Gladiator\Models\User::class, function (Faker\Generator $faker) {
+$factory->define(User::class, function (Generator $faker) {
     return [
-        'first_name' => $faker->firstName,
-        'last_name' => $faker->lastName,
-        'email' => $faker->email,
-        'password' => bcrypt(str_random(10)),
-        'remember_token' => str_random(10),
+        'id' => str_random(24),
+        'role' => null,
     ];
 });
 
-$factory->define(Gladiator\Models\Contestant::class, function (Faker\Generator $faker) {
+$factory->define(WaitingRoom::class, function (Generator $faker) {
     return [
-        'first_name' => $faker->firstName,
-        'last_name' => $faker->lastName,
-        'email' => $faker->email,
-        'dosomething_id' => $faker->numberBetween(1000, 9000),
+        'campaign_id' => $faker->numberBetween(10, 300),
+        'campaign_run_id' => $faker->numberBetween(1000, 3000),
+        'signup_start_date' => $faker->dateTimeBetween('-3 weeks', 'now'),
```

Comment:
fake data so it doesn't really matter, but +/- 3 weeks for waiting rooms seems wrong